examples/objc/main_darwin.go

```diff
@@ -17,45 +17,42 @@ var (
 	sel_bar    = objc.RegisterName("bar")
 )
 
-type barObject struct {
-	isa objc.Class `objc:"BarObject : NSObject <NSDelegateWindow>"`
-	bar int
-}
-
-func (b *barObject) Init(_cmd objc.SEL) objc.ID {
-	return objc.ID(unsafe.Pointer(b)).SendSuper(_cmd)
-}
-
-func (b *barObject) Bar(_cmd objc.SEL) int {
-	return b.bar
-}
-
-func (b *barObject) SetBar(_cmd objc.SEL, bar int) {
-	b.bar = bar
-}
-
-func (_ *barObject) Selector(metName string) objc.SEL {
-	switch metName {
-	case "Init":
-		return sel_init
-	case "SetBar":
-		return sel_setBar
-	case "Bar":
-		return sel_bar
-	default:
-		return 0
-	}
+func BarInit(id objc.ID, _cmd objc.SEL) objc.ID {
+	return id.SendSuper(_cmd)
```

Comment:
Why `_cmd` rather than `cmd`?

Reply:
It was just a holdover from the previous code. We had decided to use cmd from now on.